mscore/scoreview.cpp

```diff
@@ -1904,10 +1904,15 @@ void ScoreView::cmd(const char* s)
             }
       else if (cmd == "up-chord") {
             Element* el = score()->selection().element();
+            Element* oel = el;
             if (el && (el->isNote() || el->isRest()))
                   cmdGotoElement(score()->upAlt(el));
             el = score()->selection().element();
-            while (el->isRest() && toRest(el)->isGap() && el->voice() != 0) {
```

Comment:
Dereferencing _el_ before making sure that it is not NULL causes a crash. Also, gaps in voice 1 should not be possible. But if there were a gap in voice 1, still handle it correctly.